templates/example_cell_clustering.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>What if a user wants to run this in a different Docker session? For example, if a user first does pixel clustering and works with that data for awhile, then decides they want to do cell clustering later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@cliu72 good question. If they do, it's gonna break the pipeline since we store a lot of variables from pixel clustering and there won't be anything to recall if they open a new Docker session. Any thoughts @ngreenwald?